shapeshift/web
# fix: bump wherever to working version

## Description

Update Wherever widget to a working version with working walletconnect bridge and updated CSP.

## Pull Request Type


## Risk

Reverted feature flag, low risk.

## Testing

- Run in dev
- Make sure widget works
- Make sure constant console errors are no longer appearing

### Engineering

☝️

### Operations

☝️
